preset/classical-ml/docker-compose.yaml

```
@@ -37,6 +37,13 @@ services:
         https_proxy: ${https_proxy}
         no_proxy: ''
       context: .
+      labels:
+        docs: classical-ml
```

```suggestion
        docs: classical_ml
```
This conflicts with the base containers in `classical-ml/`